packages/protocol/test/governance/network/governance.ts

```diff
@@ -998,8 +999,28 @@ contract('Governance', (accounts: string[]) => {
         assert.isFalse(await governance.isQueued(proposalId))
         assert.equal((await governance.getQueueLength()).toNumber(), 1)
         assert.equal((await governance.dequeued(0)).toNumber(), proposalId)
+        assert.isTrue(originalLastDequeue.toNumber() < (await governance.lastDequeue()).toNumber())
```

Comment:
Could potentially use `assert.isBelow` for these inequality checks, here and elsewhere.